 Fix action bar covering bitmap in the Crop editor

forces the action bar on the editor to show some additional infos about
effects values being applied (like exposure level) but in the Crop editor
we don't have any info there and the action bar was overlapping the top crop handles

diff --git a/src/com/android/gallery3d/filtershow/FilterShowActivity.java b/src/com/android/gallery3d/filtershow/FilterShowActivity.java
index f41a979..017102a 100644
--- a/src/com/android/gallery3d/filtershow/FilterShowActivity.java
+++ b/src/com/android/gallery3d/filtershow/FilterShowActivity.java
@@ -429,8 +429,8 @@ public void loadEditorPanel(final FilterRepresentation representation) {
         if (mCurrentEditor.showsActionBar()) {
             setActionBar();
             showActionBar(true);
-        } else {
-            //showActionBar(false);
+        } else if (currentId == EditorCrop.ID) {
+            showActionBar(false);
         }
 
         if (representation.getFilterType() == FilterRepresentation.TYPE_WATERMARK_CATEGORY) {
